packages/lib-xrpl/index.ts

```diff
@@ -0,0 +1,77 @@
+import { DfnsApiClient, DfnsError } from '@dfns/sdk'
+import { Transaction, encode } from 'xrpl'
+import { GetWalletResponse, GenerateSignatureResponse } from '@dfns/sdk/types/wallets'
+import { hashSignedTx } from 'xrpl/dist/npm/utils/hashes'
```

Comment:
let's try not to import from private package path (when possible), the way to get this properly seems to be add import `hashes` from `xrpl` above. then later you can call `hashes.hashSignedTx(signedTx)`